apps/myaccount/src/components/change-password/change-password.tsx

```diff
@@ -262,10 +262,11 @@ export const ChangePassword: FunctionComponent<ChangePasswordProps> = (props: Ch
                             setPassword(values.get("newPassword").toString());
                         } }
                         validation={ (value: FormValue, validation: Validation) => {
-                            if (passwordScore < 3) {
+                            // Disable password strength validation
```

Comment:
Why is it removed? No longer required? If so shall we remove the code out?

Reply:
Resolved by ff6c234b